diff --git a/Dockerfile b/Dockerfile
index f17077c..0e3e184 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -8,7 +8,7 @@ RUN apt-get -y update && \
         git vim parted \
         quilt coreutils qemu-user-static debootstrap zerofree zip dosfstools e2fsprogs\
         libarchive-tools libcap2-bin rsync grep udev xz-utils curl xxd file kmod bc \
-        binfmt-support ca-certificates fdisk gpg pigz arch-test \
+        binfmt-support ca-certificates fdisk gpg gpgv pigz arch-test \
     && rm -rf /var/lib/apt/lists/*
 
 COPY . /pi-gen/
